src/main.rs

```diff
@@ -32,6 +32,7 @@ struct AppState {
     ethereum_client: HttpClient,
     succinct_client: Client,
     succinct_base_url: String,
+    beaconcha_url: String,
```

Comment:
Should be `beaconchain_base_url` I think `.in` doman is part of the service's name.